bioio_base/reader.py

```diff
@@ -1089,12 +1089,25 @@ def standard_metadata(self) -> StandardMetadata:
                     metadata.objective = "40x Magnification"
                     return metadata
         """
+        # Attempt to get OME metadata; ignore if not implemented
+        try:
+            ome = self.ome_metadata
+        except NotImplementedError:
+            ome = None
+
         # Retrieve the dimensions information from the reader.
         dims = self.dims
```

Comment:
Feels like this could just be removed and the references below could use `self.dims`

Reply:
resolved in [2171dc8](https://github.com/bioio-devs/bioio-base/pull/38/commits/2171dc8bf24c9ca10719acecd507c0c2f2cb4850)